templates/donation_successful.html

```diff
@@ -1,5 +1,194 @@
-<html>
+<!doctype html>
+<html lang="en" class="no-js">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<!-- Request Volkorn, Open Sans, Source Sans Pro, Signika (regular and semibold), Overlock -->
+	<link href='http://fonts.googleapis.com/css?family=Vollkorn|Open+Sans|Signika|Source+Sans+Pro:400,600,700' rel='stylesheet' type='text/css'>
+
+	<link href="https://fonts.googleapis.com/css?family=Overlock:400i" rel="stylesheet">
+
+
+	<link rel="stylesheet" href="../static/css/reset.css"> <!-- CSS reset -->
+	<link rel="stylesheet" href="../static/css/style.css"> <!-- Resource style -->
+	<link rel="stylesheet" href="../static/css/menu.css"> <!-- Resource style -->
+
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+
+	<!-- <link rel="stylesheet" href="../static/bower_components/font-awesome/css/font-awesome.min.css"> Resource style -->
```

Comment:
I'm not sure, I just copy-pasted the code over from index.html's <head>, and that's how it was. Also, there's no static/bower_components directory in the repo